juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf

```diff
@@ -0,0 +1,226 @@
+iceberg {
```

Comment:
461
Entering configuration mode
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf:223:(19) syntax error: system-settings
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf:225:(5) error recovery ignores input until this point: }
load complete (2 errors)

Reply:
seems to be error with CI not code:

Setting status of 0f874b7c1b8a8bcfa098f56d6c4c904ec6c97547 to FAILURE with url http://10.209.18.90:8081/job/iceberg/job/jfit-rules_validate/482/ and message: 'Build finished. '
ERROR: Error fetching remote repo 'origin'
Finished: FAILURE